app/_locales/en/messages.json

```diff
@@ -2564,10 +2564,13 @@
     "message": "Transaction Fee"
   },
   "transactionHistoryBaseFee": {
-    "message": "Base fee (GWEI)"
+    "message": "Base Fee (GWEI)"
```

Comment:
Opted to titlecase all of the properties in the transaction breakdown to keep things consistent.